integrationTests/testInitializer.go

```diff
@@ -1,6 +1,7 @@
 package integrationTests
 
 import (
+	"bytes"
```

Comment:
after the executingMiniblocks_test refactoring, please come back to this file and remove unneeded funcs.

Reply:
Done.